models/KelpGreenling2015/warning.sso

```diff
@@ -4,12 +4,14 @@ Warning 3 : fishing fleet: 3 surveytime read as: 1 normally is -1 for fishing fl
 Warning 4 : fishing fleet: 4 surveytime read as: 1 normally is -1 for fishing fleet; can override for indiv. obs. using 1000+month
 Warning 5 : fishing fleet: 5 surveytime read as: 1 normally is -1 for fishing fleet; can override for indiv. obs. using 1000+month
 Note 1 Suggestion: This model has just one settlement event. Changing to recr_dist_method 4 and removing the recruitment distribution parameters at the end of the MG parms section (below growth parameters) will produce identical results and simplify the model.
-Warning 6 : Fleet: 1  agecomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 105
-Warning 7 : fleet: 2  lencomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 31
-Warning 8 : Fleet: 2  agecomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 140
-Warning 9 : fleet: 3  lencomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 30
-Warning 10 : fleet: 4  lencomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 22
-Warning 11 : Fleet: 5  agecomp has obs with partition==2; will treat as partition=0 because retention not defined; N= 8
-Warning 12 : Forecast F capped by max possible F from control file4
+Warning 6 : At least one block pattern ends in endyr. Check the output parameter value time series to see if the values in forecast years are as intended.
+Warning 7 Adjustment: Fleet: 1  agecomp has N obs with partition==2 (retained); changed to partition=0 because retention not defined; N= 105
```

Comment:
@Rick-Methot-NOAA Should we change the partitions in this model for the length compositions to just 0 so this error disappears in the warning.sso?